wave_kinematics/include/wave/kinematics/rotation.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef __WAVE_KINEMATICS_ROTATION_HPP__
+#define __WAVE_KINEMATICS_ROTATION_HPP__
+
+#include <iostream>
+#include <functional>
+
+#include "wave/utils/utils.hpp"
+#include <kindr/Core>
+
```

Comment:
I tend to folow a header file order of: 

1. C headers
2. C++ headers
3. External library headers
4. Internatl library headers